setup.go

```diff
@@ -110,6 +110,19 @@ func parse(c *caddy.Controller) (*Gateway, error) {
 					return nil, c.Errf("ttl must be in range [0, 3600]: %d", t)
 				}
 				gw.ttlLow = uint32(t)
+			case "negttl":
```

Comment:
`negttl` block is the same as `ttl`

We can do something like this:
```go
			case "negttl","ttl": // <--- 
				args := c.RemainingArgs()
				if len(args) == 0 {
					return nil, c.ArgErr()
				}
				t, err := strconv.Atoi(args[0])
				if err != nil {
					return nil, err
				}
				if t < 0 || t > 3600 {
					return nil, c.Errf("%s must be in range [0, 3600]: %d", c.Val(), t)
				}
				gw.ttlHigh = uint32(t)
```

Reply:
refactored